.github/workflows/build.yml

```diff
@@ -13,10 +13,10 @@ jobs:
 
     steps:
     - name: clone
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
```

Comment:
use exact version in comments like we do on core repos: https://github.com/go-vela/server/blob/9bb4acca095dc88f92c48dd45f9bdd587740115b/.github/workflows/build.yml#L16 (for all the actions)